addons/rook/template/testgrid/k8s-docker.yaml

```diff
@@ -15,9 +15,8 @@
     rook:
       version: "__testver__"
       s3Override: "__testdist__"
-      isBlockStorageEnabled: true
```

Comment:
Are we removing this flag because it's already set to `true` by default?

Reply:
yes. we dont even use the variable in rook versions > 1.0.4